document-generator-api/src/main/java/uk/gov/companieshouse/document/generator/api/models/DocumentRequest.java

```diff
@@ -11,7 +11,10 @@
     @JsonProperty("resource_uri")
     private String resourceUri;
 
-    @NotNull
```

Comment:
Shouldn't the NOT_NULL annotation still exist until this field is removed? Even though it's deprecated it should still have the same behaviour to make sure existing code doesn't break.

Reply:
it's not being used any more, so all reference to it has been removed from the api side. so no getters call the parameter.